### DISCUSSION TITLE
AGS v2 push

### DISCUSSION THREAD

**Discussion by UcraNerd** [end-4/dots-hyprland#946] [General]
So, for those who are wondering what I'm yapping about here the official discussion of the AGS project: https://github.com/Aylur/ags/discussions/615; then let's go on, @end-4 what are your plans for this sudden version push? As the AGS' maintainer says, `AGS v1 has been pretty much abandoned for months now.` and as some users say in the discussion, the configs need to be recreated from scratch. I really love these dot files, so I would appreciate some feedbacks about this situation.

**Comment by UcraNerd** [comment]
For those who might have installed the v2 package from AUR, you can download the PKGBUILD file from https://github.com/kotontrion/PKGBUILDS/blob/main/agsv1/PKGBUILD and run `makepkg -si` after changing directory to the directory with the downloaded file; to uninstall you can use pacman. 

> Make shure to edit the package name from agsv1 to ags, unless you want to change every single reference to ags in the configs, like I did.

**Comment by clsty** [comment]
Well, just happened to see this thread... Seems not an easy task.

Issue already created #1092.
